src/up.ts

```diff
@@ -68,8 +77,12 @@ const go = async function () {
 
 // start a tableland network, then catch any uncaught errors and exit loudly
 go().catch((err) => {
-  console.error("unrecoverable error");
-  console.error(err);
+  if (err.message.includes(`port already in use`)) {
+    console.error(err.message);
```

Comment:
Sorry for the nit here, but it would be nice to log the entire error object.  It makes debugging unexpected errors more difficult if you don't have a stack trace.

Reply:
i'm all for the nits! updated them.